src/main/java/net/consensys/orion/http/handler/receive/ReceiveHandler.java

```diff
@@ -79,12 +83,18 @@ public void handle(final RoutingContext routingContext) {
         return;
       }
 
-      final byte[] decryptedPayload;
-      try {
-        decryptedPayload = enclave.decrypt(encryptedPayload.get(), recipient);
-      } catch (final EnclaveException e) {
-        log.info("unable to decrypt payload with key {}", key);
-        routingContext.fail(404, new OrionException(OrionErrorCode.ENCLAVE_KEY_CANNOT_DECRYPT_PAYLOAD, e));
+      byte[] decryptedPayload = null;
```

Comment:
final

Reply:
It doesn't let me, because it is assigned in the loop, even though I'm only doing it once ...